### ISSUE TITLE
[Feature Request] Option to always show content in original language

### ISSUE THREAD

**Issue by neneko** [TeamNewPipe/NewPipe#3185]
The search results always seem to display video titles in their original language, but then changes to English when I go to view them. There is an option to change content language (Settings->Content->Default content language) but this isn't really what I want, since I want videos where the title name is in English originally to still be in English.

**Comment by B0pol** [comment]
> since I want videos where the title name is in English originally to still be in English.

I don't really understand. You may not know, but this selector is broken for 5 months, and it has been set to English only.
Then for content originally in English, the language should be English in the current NewPipe version, even if it was translated to other languages.
If it's not the case, please link a video / channel where is doesn't occur.
For videos originally NOT in English and translated to English, indeed titles / descriptions are shown in English if the video is translated, because as I said for 5 months English only has been set.

But your feature request is about original language, and that's possible because youtube provides two descriptions, two titles… : original language and desired language.

**Comment by B0pol** [comment]
Duplicate of #3089

**Comment by realmain** [comment]
I am really hoping that this gets updated in the future. It's really annoying if I get into a video with a Vietnamese title, it auto translate it to English.

**Comment by ggdorman** [comment]
Yes, the automatic translation is rather frustrating. Hopefully this can be fixed.
